src/ir/LocalGraph.cpp

```diff
@@ -302,12 +305,8 @@ struct LocalGraphFlower
   // initially, but instead fill in these data structures that let us do so
   // later for individual gets. Specifically we need to find the location of a
   // local.get in the CFG.
-  struct BlockLocation {
-    // The basic block an item is in.
-    FlowBlock* block = nullptr;
-    // The index in that block that the item is at.
-    Index index;
-  };
+  using BlockLocation = std::pair<FlowBlock*, Index>;
```

Comment:
Why this change? It seems that having the custom field names would be better.

Reply:
The names are nice, but in practice this struct is always immediately destructured,
```cpp
auto [block, index] = ..
```
so we are not losing names here. And, using a `std::pair` gives us hashing for free without writing new boilerplate, which is needed in this PR (it uses a map on this type).